manifests/charts/istiod-remote/files/injection-template.yaml

```diff
@@ -238,7 +238,7 @@ spec:
           - drain
   {{- end }}
     env:
-    {{- if eq (env "PILOT_ENABLE_INBOUND_PASSTHROUGH" "true") "false" }}
+    {{- if eq (env .InboundTrafficPolicyMode "localhost") "passthrough" }}
```

Comment:
Hmmm I did not realize this would require us to maintain support in the injection template. This is kind of messy and implies this should be a proxy config setting instead of mesh config?

Reply:
I am fine changing/moving this proxy config. But it typically applies to all proxies though. Should I change to proxyConfig?

Reply:
yes, donot change that env

Reply:
We just need to use the InboundTrafficPolicyMode during generating inbound endpoint, so we don't need per proxy setting since it is a mesh global setting